doc/whats_new/v1.2.rst

```diff
@@ -280,6 +280,11 @@ Changelog
   :pr:`22710` by :user:`Conroy Trinh <trinhcon>` and
   :pr:`23461` by :user:`Meekail Zain <micky774>`.
 
+- |Fix| :func:`metrics.logloss` takes "auto" as default eps value and it will be equal to
+  eps value of the `y_pred` if `y_pred` is numpy float else it will be 1e-15. This change was
+  made to be able to handle float16 and float32 numpy arrays
+  :pr:`24354` by :user:`Safiuddin Khaja <Safikh>`
```

Comment:
COuld you also acknowledge @gsiisg since he also open and reported the error?
It is a bit unfortunate that we have 2 PRs.